src/docs/training.md

```diff
@@ -0,0 +1,27 @@
+## SSSOM Training materials
+
+- [Elevator pitch](#elevator)
+- [Tutorials and Guides](#guides)
+- [Related tutorials](#related)
+
+<a id="elevator"></a>
+
+### Elevator pitch
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/T-YP-Lv9k2s?si=tN9lHUFcovkyvm6z" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
A 11-minutes video? What kind of elevator are you taking? :D

Reply:
Hahah. You are right of course. There are some elevators in Athens that take 11 minutes to go 5 stories but that is not what the general public would understand :P